pytket/extensions/iqm/backends/iqm.py

```diff
@@ -90,8 +86,7 @@ class IQMBackend(Backend):
     def __init__(
         self,
         url: str,
-        settings: PathLike,
-        arch: Optional[List[Tuple[str, str]]] = None,
+        arch: Optional[List[List[str]]] = None,
```

Comment:
Why changed from pair to list?

Reply:
If `None`, this parameter is overwritten by a list retrieved from the server (few lines later):

```python
_iqmqa = self._client.get_quantum_architecture()
...
if arch is None:
            arch = _iqmqa.qubit_connectivity
``` 

`_iqmqa.qubit_connectivity` is a list of lists, so I changed the type signature of the function parameter accordingly. This satisfies the static type checker. 

Otherwise, the retrieved list of lists has to be converted to a list of tuples. I thought it does not make sense to adapt to the existent type of a parameter (a parameter, which normally is not supposed to be used).

Reply:
Makes sense, thanks.